src/js/HuesCore.ts

```diff
@@ -1162,85 +1162,83 @@ export class HuesCore extends EventListener<CoreEvents> {
         }
     }
 
-    keyHandler(key: number) {
+    keyHandler(key: string) {
         switch (key) {
-        case 37: // LEFT
+        case "ArrowLeft":
             this.previousImage();
             break;
-        case 39: // RIGHT
+        case "ArrowRight":
             this.nextImage();
             break;
-        case 38: // UP
+        case "ArrowUp":
+        case "MediaTrackNext":
             this.nextSong();
             break;
-        case 40: // DOWN
+        case "ArrowDown":
+        case "MediaTrackPrevious":
             this.previousSong();
             break;
-        case 70: // F
+        case "f":
             this.toggleFullAuto();
             break;
-        case 109: // NUMPAD_SUBTRACT
-        case 189: // MINUS
-        case 173: // MINUS, legacy
+        case "-":
             this.soundManager.decreaseVolume();
             break;
-        case 107: // NUMPAD_ADD
-        case 187: // EQUAL
-        case 61: // EQUAL, legacy
+        case "=": // actualy a "+" but shift is disabled
```

Comment:
Why can't we do something like `e.shiftKey`?

Reply:
In the calling function shiftKey is intercepted:

https://github.com/mon/0x40-web/blob/694c6fd4261929b6a60d18e0ae023349f8259e6b/src/js/HuesCore.ts#L335-L337

even if it wasn't, the case would still end up with "=", the event for typing "+" would look like `{.key="=", .shiftKey=true}`

Was just trying to maintain existing behaviour, currently `=` works but `+` doesn't for volume control on https://0x40.mon.im